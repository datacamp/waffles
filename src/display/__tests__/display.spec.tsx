import { render } from '@testing-library/react';

import { Display } from '../index';

describe('Display', () => {
  it('renders a paragraph containing the text', () => {
    const { container } = render(<Display>We are bold!</Display>);

    const displayParagraph = container.querySelector('p');

    expect(displayParagraph).toBeInTheDocument();
    expect(displayParagraph).toHaveTextContent('We are bold!');
  });

  it('renders snapshot', () => {
    const { getByText } = render(<Display>Test</Display>);

    const displayParagraph = getByText('Test');

    expect(displayParagraph).toBeInTheDocument();
    expect(displayParagraph).toMatchSnapshot();
  });
});
