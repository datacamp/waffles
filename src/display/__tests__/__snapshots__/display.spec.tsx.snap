// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display renders snapshot 1`] = `
.emotion-0 {
  color: #05192D;
  font-family: Studio-Feixen-Sans,Arial,sans-serif;
  font-size: 14px;
  font-weight: 400;
  line-height: 1;
  margin: 0;
  padding: 0;
  font-size: 28px;
  font-weight: 800;
  line-height: 1.25;
  letter-spacing: -0.5px;
}

<p
  class="emotion-0"
>
  Test
</p>
`;
